contracts/ERC7432/ERC7432.sol

```diff
@@ -81,16 +83,19 @@ contract ERC7432 is IERC7432 {
         address _revoker,
         address _grantee
     ) external override onlyApproved(_tokenAddress, _tokenId, _revoker) {
```

Comment:
Should we modify the `onlyApproved` modifier to allow approved accounts to revoke on behalf of the `_grantee` as well? Since the `_grantee` can call `revokeRole` as well